packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/ClusterManagersController.java

```diff
@@ -225,6 +256,35 @@ protected void onClusterItemRendered(@NonNull T item, @NonNull Marker marker) {
     }
   }
 
+  /** AdvancedMarkerClusterRenderer is a ClusterRenderer that supports AdvancedMarkers */
+  @VisibleForTesting
+  static class AdvancedMarkerClusterRenderer<T extends MarkerBuilder>
```

Comment:
Does this need to be a different class? If it does can it extend MarkerClusterRenderer? 

I worry about keeping the two in sync.

Reply:
Yes, it does. `android-maps-utils` have two different renderers - one for each marker type (legacy and advanced). This new class extends [DefaultAdvancedMarkersClusterRenderer](http://googlemaps.github.io/android-maps-utils/android-maps-utils%20/com.google.maps.android.clustering.view/-default-advanced-markers-cluster-renderer/index.html) which is meant to be used with `AdvancedMarker`s. As far as I remember, `android-maps-utils` recreates cluster markers under the hood so it has to know which marker type to use